configurations/gefion1_FakeDINI+FakeIFS/datastore.yaml

```diff
@@ -32,7 +32,8 @@ output:
 
 inputs:
   danra_sl_state:
-    path: /harmonie_cy40/danra/w12p05_s45p65_e24p52_n64p40/dx2p5km_dy2p5km/<analysis_time>/single_levels.zarr/
+    # path: /dcai/projects/cu_0003/data/sources/danra/v0.5.0/single_levels.zarr/
+    path: https://object-store.os-api.cci1.ecmwf.int/danra/v0.5.0/single_levels.zarr
```

Comment:
the original should be changed, you are right about that. But does this contradict that we wanted to keep the original?

Reply:
ah sorry, this shouldn't be in this PR. Will revert